### DISCUSSION TITLE
What's the purpose of ParableInstruments plugin?

### DISCUSSION THREAD

**Discussion by RCJacH** [DISTRHO/Cardinal#418] [Q&A]
I came across this plugin and noticed that almost every file in the repo points to the ArableInstruments plugin, which is also compiled as a separate plugin. What is the purpose of the Parable Instruments plugin then?

**Comment by falkTX** [comment]
Moved to discussions.

They are setup in a weird way because the original sources use the same repo with different branches for the different modules.
So in order to reduce build complexity and size, cardinal reuses the same files for the 2 plugins.
